src/dataMigration.js

```diff
@@ -73,6 +73,27 @@ const dataMigrations = [
           }
         });
       });
+
+      const nameResults = database.objects('Name').filtered('id = $0',
+                                settings.get(SETTINGS_KEYS.SUPPLYING_STORE_NAME_ID));
+      if (nameResults.length < 1) throw new Error('Supplying Store Name ID missing from settings');
+      const mainSupplyingStoreName = nameResults[0];
+
+      const requisitions = database.objects('Requisition')
+                                   .filtered('otherStoreName = Null').snapshot();
```

Comment:
Thanks, nice catch, updated.
As per capital Null, I didn't want to go deep into research, just found the following (since latest realm js docs didn't include undefined/null value search)
https://github.com/realm/realm-js/issues/162 , then pr file changes https://github.com/realm/realm-js/pull/386/files there an enum .. I didn't know if case sensitive
 `enum class Type { None, Number, String, KeyPath, Argument, True, False, Null } type; `, will check with = null and == null tomorrow